src/data/key-configs.js

```diff
@@ -1,4 +1,5 @@
 /**
+ * @flow
```

Comment:
Nit - can you move `// @flow` outside of the docstring? Just for consistency. I'm being really nitty-picky here :D